beater/request/context.go

```diff
@@ -91,7 +93,7 @@ func NewContext() *Context {
 // Reset allows to reuse a context by removing all request specific information.
 //
 // It is valid to call Reset(nil, nil), which will just clear all information.
-// If w and r are non-nil, the context will be associated with them for handling
+// If W and r are non-nil, the context will be associated with them for handling
```

Comment:
```suggestion
// If w and r are non-nil, the context will be associated with them for handling
```

w here refers to the parameter name, not the field name

Reply:
Resolved in 5fdff5d11